### ISSUE TITLE
Suggestion: vim-like use of numbers

### ISSUE THREAD

**Issue by jonathandturner** [jmacdonald/amp#105]
I noticed we aren't using number keys for much (anything?) yet. Amp could potentially use them the same way you might in vi/vim: to repeat the following instruction.

Example: Highlighting this row and the next three could be `V`, `3`, `down arrow`.

**Comment by chris-ricketts** [comment]
I agree - coming from vi/vim I mainly use this to yank multiple lines and/or copy a register x times. 
In amp it could be `3y` then `3p`
